composer.json

```diff
@@ -16,13 +16,13 @@
 	},
 	"require-dev": {
 		"automattic/vipwpcs": "^2.2",
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7",
+		"dealerdirect/phpcodesniffer-composer-installer": "^1.0",
 		"php-parallel-lint/php-parallel-lint": "^1.0",
 		"phpcompatibility/phpcompatibility-wp": "^2.1",
-		"phpunit/phpunit": "^4 || ^5 || ^6 || ^7",
+		"phpunit/phpunit": "^9",
```

Comment:
This plugin claims to work with PHP 5.6 and above, but this change now limits testing to only on PHP 7.3 and above.

Reply:
Ahh you're right, I'll update that, as even 8.0 is nearing EoL

Reply:
https://github.com/Automattic/msm-sitemap/pull/175

Reply:
I don't think updating the minimum supported version is the way to go here. As much as I'd like to use newer syntax, there's no user benefit to us excluding users on earlier versions of PHP. 

At least not without a major version release.

Reply:
I don't feel like allowing usage on 5.4 is the way forward, 5.4 has been EoL for almost 8 years